qdrant-landing/content/documentation/private-cloud/private-cloud-setup.md

```diff
@@ -91,5 +94,6 @@ To uninstall the Qdrant Private Cloud solution, you can use the following comman
 
 ```bash
 helm uninstall qdrant-private-cloud --namespace qdrant-private-cloud
+helm uninstall qdrant-private-cloud-crd --namespace qdrant-private-cloud
```

Comment:
Missing `s`? `qdrant-private-cloud-crds`

Reply:
This is going to remove all `QdrantCluster` instances, perhaps should we communicate that to users?